Application/EdFi.Ods.Api/Controllers/TokenController.cs

```diff
@@ -5,6 +5,8 @@
 
 #if NETCOREAPP
 using System;
+using System.Net;
+using System.Net.Http;
```

Comment:
this should not be here for a net core application

Reply:
Fixed